### ISSUE TITLE
Make all templates consistent

### ISSUE THREAD

**Issue by sayedihashimi** [dotnet/dotnet-template-samples#29]
See https://github.com/dotnet/dotnet-template-samples/pull/20#issuecomment-330354670

We should update the templates to use the same naming patterns.

**Comment by guardrex** [comment]
Cross-ref for the topic repo work that goes along with this: https://github.com/dotnet/docs/issues/3175

**Comment by guardrex** [comment]
@sayedihashimi I'm asking [@]mairaw if she'll let me take care of the topic. If she says :+1:, I'll need to know what name you (or your assignee here) would like to use.

* Is there anything else outside of the `identity` and `shortname` to change?
* What values would you like to use for `identity` and `shortname`?

I'll track this and be on stand-by until more is known.

**Comment by neman** [comment]
I agree that we should define values for shared (common) variables used across project examples. I looked a little bit more, and found that there are inconsistencies even among first 15 examples. I will post a detailed info soon.

**Comment by sayedihashimi** [comment]
Thanks @neman
